README.md

```diff
@@ -191,4 +191,7 @@ New versions of the package registry need to be released from time to time. The
 
 CI automatically creates a new Docker image which will be available under `docker.elastic.co/package-registry/package-registry:vA.B.C` a few minutes after creating the tag.
 
-As a new registry is normally released to bring new features to the package-storage, follow the docs in the [Package Storage](https://github.com/elastic/package-storage#update-package-registry-for-a-distribution) repository on how to update the distributions.
+After the new registry Docker image is available, update the following projects that consume it:
+- Integrations: Update the version of the Package Registry Docker image as shown in this [sample PR](https://github.com/elastic/integrations/pull/581).
+- Package Storage: Follow the [documentation](https://github.com/elastic/package-storage#update-package-registry-for-a-distribution) on how to update the distributions.
+- Kibana: Do this only after all Package Storage distributions have been updated and released as new Docker images. Update the version of the Package Registry Distribution Docker image as shown in this [sample PR](https://github.com/elastic/kibana/pull/89776).
```

Comment:
Good question on how often we should update these? Each time a stack release goes out, we should tag a version of the package-storage which then also can be used in Kibana. This makes it more readable. Unfortunately we didn't do a great job at this so far :-(

Reply:
What do you think about changing this dependency in Kibana to look at the latest one? We don't expect any breaking changes on the EPR side.

Reply:
I quite like tying it to a specific version and then update is manually. This is especially true for the release branches where the testing should be fully predictable.

Reply:
Yeah, but Kibana depends on the package-storage, not package-registry. We don't have a consistent versioning there and we don't need it.

Reply:
> We don't have a consistent versioning there and we don't need it.

As mentioned above, I think we should have tags for the production ones. Not for the release itself but exactly for testing or running on prem in the future.

Reply:
Just so I'm super clear, @ruflin, you're referring to adding another Docker tag on the `package-storage` image besides `snapshot`, `production`, etc. that refers to the `package-registry` version that image was built with, correct?

If so, it looks like we already add a *label* for this information: https://github.com/elastic/package-storage/blob/5314869e2f6bc01d37b8652f7bda89248950b3a4/Dockerfile#L8

But since images can't be pulled down using labels, it's just a matter of also adding the same label as a tag as well, e.g. `snapshot-0.17.0`, `production-0.17.0`, etc.?

Reply:
No :-) What I'm looking for is a tag that is aligned with the stack release like we have for 7.9.0: https://github.com/elastic/package-storage/tree/7.9.0 So this means, we should have created a tag `7.11.0` on the day of the 7.11 Elastic Stack release. This is documented here: https://github.com/elastic/package-storage#tags

Reply:
Ah okay, thanks for clarifying :) So this will need to be a completely manual process or can we make it part of the release manager CI job?

Reply:
Would be great to have it automated. How are tags in other repos created?

Reply:
I've created an infra issue to ask about automating this and tagged you both on it.

Reply:
@ycombinator I think there are already issues and @kuisathaverat is working on this already.

Reply:
> Would be great to have it automated. How are tags in other repos created?

It depends, we have two kinds:
* The tag is created manually on the repo and it triggers a release process
* A job is triggered in the CI that creates a tag and make a release process

We are working on automerge the package-storage PRs and trigger the deployment automatically https://github.com/elastic/package-storage/issues/592, I think is not the same.

Reply:
@ycombinator @kuisathaverat Sorry, I mixed two things here. You are right.

Reply:
I'm going to merge this PR with what we have here. If/when we have automation, we can always come back to this README and remove the `Kibana` line or update it to reflect the reality of that time.